packages/server/utils/__tests__/isRequestToJoinDomainAllowed.test.ts

```diff
@@ -51,7 +51,7 @@ const addOrg = async (
   const org = {
     id: orgId,
     activeDomain,
-    featureFlags: featureFlags ?? []
```

Comment:
`featureFlags` is completely absent in the db when no flags are set for an org. In the tests I set it to `[]`, that's why the tests passed previously.